src/components/results/shortcircuit/shortcircuit-analysis-result-table.tsx

```diff
@@ -29,15 +31,14 @@ import {
 import { useSelector } from 'react-redux';
 import { ComputingType } from '../../computing-status/computing-type';
 import { ReduxState } from '../../../redux/reducer.type';
-import CustomHeaderComponent from '../../custom-aggrid/custom-aggrid-header';
-import { ISortConfig } from '../../../hooks/use-aggrid-sort';
 import { DefaultCellRenderer } from '../../spreadsheet/utils/cell-renderers';
+import { DATA_KEY_TO_SORT_KEY } from 'components/results/shortcircuit/shortcircuit-analysis-result-content';
 
 interface ShortCircuitAnalysisResultProps {
-    result: SCAResultFault[];
-    onSortChanged: (colKey: string, sortWay: number) => void;
-    sortConfig: ISortConfig;
-    analysisType: ShortcircuitAnalysisType;
+    result: SCAFaultResult[];
+    analysisType: ShortCircuitAnalysisType;
+    updateFilter: (filter: any) => void;
+    updateSort: (sort: any) => void;
```

Comment:
You can create objects for both filtering and sorting to replace the any type. This approach is clearer

Reply:
Yes you're right I will add it